rs/sns/testing/README.md

```diff
@@ -129,35 +180,50 @@ To re-run it, remove `$PWD/.dfx` directory to be able to re-deploy the canister
 
 ## Interact with SNS lifecycle
 
-This section provides a step-by-step guide on how to deploy a user-defined SNS and interact with it during its lifecycle.
+This section provides a step-by-step guide on how to deploy a user-defined SNS and interact with it
+during its lifecycle.
 
 ### Getting ICP tokens
 
-You can use `sns-testing --network http://127.0.0.1:8080 transfer-icp` to get ICP tokens to your account.
+You can use `sns-testing --network http://127.0.0.1:8080 transfer-icp` to get ICP tokens to your
+account.
 
 This subcommand supports transfer to the direct account ID, e.g.:
 ```
-sns-testing --network http://127.0.0.1:8080 transfer-icp --amount 650.0 --to 5c9b28f3e2218975ea76449cf9b949a860d741a3af7dd4dd062f7481e3a99cde
+./sns-testing \
+   --network http://127.0.0.1:8080 \
+   transfer-icp \
```

Comment:
NCR, since the issue predates this PR.

Does this really move ICP from one balance to another? Or does this actually mint ICP? If the latter, it would be less confusing if the name contained "mint" instead of "transfer".